mirrord-config/src/lib.rs

```diff
@@ -22,39 +22,52 @@ use crate::{
     pod::PodFileConfig, util::VecOrSingle,
 };
 
+// TODO(alex) [mid] 2022-10-27: Add example usage of each field.
+
 /// This is the root struct for mirrord-layer's configuration
 #[derive(MirrordConfig, Deserialize, Default, PartialEq, Eq, Clone, Debug)]
 #[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
 #[serde(deny_unknown_fields)]
 #[config(map_to = LayerConfig)]
 pub struct LayerFileConfig {
+    /// Controls whether or not mirrord accepts invalid TLS certificates (e.g. self-signed
+    /// certificates).
     #[config(env = "MIRRORD_ACCEPT_INVALID_CERTIFICATES", default = "false")]
     pub accept_invalid_certificates: Option<bool>,
 
+    /// Allows mirrord to skip unwanted processes.
+    ///
+    /// Useful when process A spawns process B, and the user wants mirrord to operate only on
+    /// process B.
     #[config(env = "MIRRORD_SKIP_PROCESSES")]
     pub skip_processes: Option<VecOrSingle<String>>,
 
+    /// The target that will be impersonated by mirrord.
+    ///
+    /// Supports `pod`, `podname`, `deployment`, `container`, `containername`.
```

Comment:
not entirely true, you can't just use `container/name`. This needs to be better explained

Reply:
it doesn't support providing just a container
It's
`pod/{sample-pod}/[container]/container`
`deployment/{sample-deployment}/[container]/container`

Reply:
```rust
/// Supports:
/// - `pod/{sample-pod}/[container]/{sample-container}`;
/// - `podname/{sample-pod}/[container]/{sample-container}`;
/// - `deployment/{sample-deployment}/[container]/{sample-container}`;
```

like so?

Reply:
yup